test/cbmc/include/README.md

```diff
@@ -0,0 +1 @@
+../aws-templates-for-cbmc-proofs/template-for-repository/include/README.md
```

Comment:
This README file seems like the only file in this directory. What is the purpose of the directory? I see that this is the filepath to the submodule, but it may be good to add a description here as well. Also, if we aren't going to require this directory, then we should remove it.

Reply:
This include directory is only used to store any header files which are required by the test harness file. Since there are no header files included as of now, I have removed the following directory